Dockerfile-10.1.amd64

```diff
@@ -28,6 +27,6 @@ EXPOSE 3306 4444 4567 4567/udp 4568 8080 8081
 
 HEALTHCHECK CMD /usr/local/bin/healthcheck.sh
 
-ENV SST_METHOD=xtrabackup-v2
+ENV SST_METHOD=mariabackup
```

Comment:
Is 10.1 tested with mariabackup or is this just a side effect of the template? Perhaps the SST_METHOD should also be "templatized"?

Reply:
I might have forgotten to templatize it, but I also remembered having problems with the xtrabackup-v2 on the raspberry on the jsurf image. I just tried to set it up again and indeed got the following error:

```
WSREP_SST: [ERROR] FATAL: The innobackupex version is 1.5.1. Needs xtrabackup-2.3.5 or higher to perform SST (20190627 21:21:54.872)

2019-06-27 21:21:54 1715467072 [ERROR] WSREP: Failed to read 'ready <addr>' from: wsrep_sst_xtrabackup-v2 --role 'joiner' --address '10.0.38.9' --datadir '/var/lib/mysql/'   --parent '43'  ''  ''

	Read: '(null)'

2019-06-27 21:21:54 1715467072 [ERROR] WSREP: Process completed with error: wsrep_sst_xtrabackup-v2 --role 'joiner' --address '10.0.38.9' --datadir '/var/lib/mysql/'   --parent '43'  ''  '': 2 (No such file or directory)

2019-06-27 21:21:54 1963361088 [ERROR] WSREP: Failed to prepare for 'xtrabackup-v2' SST. Unrecoverable.

2019-06-27 21:21:54 1963361088 [ERROR] Aborting
```

I tried using the mariabackup, even though I used 10.1 and it just worked for me. I checked the [mariabackup page](https://mariadb.com/kb/en/library/mariabackup-overview/) and they state that it's available from 10.1.
I don't know if barely succeeding through the startup phase is enough for being considered as "tested". Anyways, I don't know how to properly test it, so I can for sure templatize it too just to not change what's working.

One other option is finding success with xtrabackup-v2 with the linuxserver base image, with which I spend the whole day today and couldn't get it to startup yet...